package.json

```diff
@@ -96,7 +96,9 @@
     "record-build-error": "cross-env ts-node --swc scripts/record-build-error.ts",
     "record-build-success": "cross-env ts-node --swc scripts/record-build-success",
     "updateProject": "cross-env ts-node --swc scripts/update-project.ts",
-    "updateCronjobImage": "cross-env ts-node --swc scripts/update-cronjob-image.ts"
+    "updateCronjobImage": "cross-env ts-node --swc scripts/update-cronjob-image.ts",
+    "delete-temp-file": "cross-env ts-node --swc scripts/delete-temp-files.ts",
+    "tempfile-deletion-Cronjob": "cross-env ts-node --swc scripts/delete-tempfile-cron.ts"
```

Comment:
Whats the purpose of these scripts?

Reply:
These last two scripts are for the another task. I added this for scheduling cronjob to remove temp files. I'm working on it.

Reply:
Removed.

Reply:
Please either get the cronjob-related commits out of the branch history, or squash everything down to one commit that doesn't have those changes anywhere in its history. Also, delete-temp-file and tempfile-deletion-Cronjob should not be in this package.json file since it's part of the cronjob branch.